src/opencl/image_function_opencl.cpp

```diff
@@ -292,6 +293,25 @@ namespace
             }
         }
 
+        __kernel void setPixelOpenCL( __global const uchar * data, uint offset, uint rowSize, uint width, uint height, uint value )
+        {
+            const size_t x = get_global_id(0);
+            const size_t y = get_global_id(1);
+
+            if ( x < width && y < height ) {
+                in[offset + y * rowSize + x] = value
+            }
+        }
+
+        __kernel void setPixelOpenCL( __global const uchar * data, volatile __global uint * offsets, uint rowSize, uint width, uint height,
```

Comment:
`offset` is only one and for image. Please use the same type as in above function `uint offset`

Reply:
changed offset